score-xml/src/main/java/io/cloudslang/content/xml/utils/ResultUtils.java

```diff
@@ -8,22 +10,24 @@
 public class ResultUtils {
     private ResultUtils(){}
 
-    public static void populateFailureResult(Map<String, String> result, String returnResult){
-        populateResult(result, Constants.FAILURE, returnResult, Constants.EMPTY_STRING, Constants.OutputNames.RESULT_XML);
+    public static void populateFailureResult(Map<String, String> result, String errorMessage){
```

Comment:
it is a good practice to format the code correctly, like placing a space between ) and {.